### PR TITLE
[receiver/dotnetdiagnosticsreceiver] fix panic when using .net6

### PR DESCRIPTION
**Pull request by yuuhhe**
**Description:** <Describe what has changed.>

keywords value is wrong

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventkeywords
https://github.com/dotnet/diagnostics/blob/main/src/Tools/dotnet-counters/CounterMonitor.cs#L774

### CODE REVIEW

File: receiver/dotnetdiagnosticsreceiver/dotnet/request.go [hunk 1]
```diff
@@ -140,14 +140,14 @@ func createProviders(intervalSec int, names ...string) (ps []provider) {
 }
 
 // from https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventlevel
-const verboseEventLevel = 5
+const verboseEventLevel = 2
```
Comment (codeboten) [inline receiver/dotnetdiagnosticsreceiver/dotnet/request.go:143]: Is this change needed?

**Review by codeboten — COMMENTED** [review]
Thanks for the contribution, the change looks ok outside of the one question. Pinging code owners @pmcollins @davmason to review as well

**Review by codeboten — COMMENTED** [review]
Please remove the `math` import to fix linting

File: receiver/dotnetdiagnosticsreceiver/dotnet/request.go (hunk 1 above)
Comment (yuuhhe) [reply receiver/dotnetdiagnosticsreceiver/dotnet/request.go:143]: It's not necessary. 
The dotnet-counters source code is `EventLevel.Error`. https://github.com/dotnet/diagnostics/blob/main/src/Tools/dotnet-counters/CounterMonitor.cs#L774

**Comment by yuuhhe** [comment]
@pmcollins @davmason please review the code

**Comment by yuuhhe** [comment]
> This PR was marked stale due to lack of activity. It will be closed in 14 days.

Who can coming to review the code?

Or what can I do? @codeboten

**Comment by codeboten** [comment]
> > This PR was marked stale due to lack of activity. It will be closed in 14 days.
> 
> Who can coming to review the code?
> 
> Or what can I do? @codeboten

@yuuhhe thanks for your patience, I've opened an issue to figure out what to do next. The guidelines for components whose owners aren't involved in the project is to mark them as unmaintained in hope other contributors volunteer to take over as code owners.
